src/simple_examples/explore_me.cpp

```
@@ -19,16 +19,26 @@ void ExploreSimpleChecks(int a, int b, std::string c) {
   }
 }
 
+#include &lt;string>
```

Cppcheck (2.18.0)

[error] No header in #include

(preprocessorErrorDirective)